ballerina/README.md

```
@@ -55,3 +76,82 @@ The JSON I/O APIs provide the reading and writing APIs for JSON content. Those A
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.   
+1. Export Github Personal access token with read package permissions as follows,
+        
+        export packageUser=<Username>
+        export packagePAT=<Personal access token>
+
+### Build the source
+
+Execute the commands below to build from source.
+
+1. To build the library:
+   ```    
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)